FHIR/src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/AnonymizerEngineTests.cs

```diff
@@ -32,6 +35,17 @@ public void GivenIsPrettyOutputSetFalse_WhenAnonymizeJson_OneLineJsonOutputShoul
             Assert.Equal(OneLineOutputTarget, result);
         }
 
+        [Fact]
+        public void GivenADicomProcessorFactory_AddingCustomProcessor_GivenMethod_CorrectProcessorWillBeReturned()
```

Comment:
Typos in the function name `GivenADicomProcessorFactory`.

Reply:
Thanks, fixed it.